src/Router.tsx

```diff
@@ -8,25 +8,24 @@ import NotFound from "components/not-found";
 import Index from "components/index";
 import ChangeNetwork from "components/change-network/ChangeNetwork";
 import useConnection from "hooks/useConnection";
-import useClient from "hooks/useOracleClient";
-import useRequestParams from "hooks/useRequestParams";
+import { forEach } from "helpers/oracleClient";
```

Comment:
added a utility to loop over all clients. since each client is independent, user state is managed independently. This means you need to update all clients on a wallet or network change.